### PR TITLE
Update payload format

### LINKED ISSUE #21: Update PN payload structure
- Remove the `alert` key.
- Update README accrdingly.

### PR DESCRIPTION
**Pull request by jaybhoyar**
Closes #21 
Ref: https://github.com/bigbinary/neeto-notifications-web/issues/32

### CODE REVIEW

**Comment by jaybhoyar** [comment]
@chiraggshah _a Please review.

File: README.md [hunk 1]
```diff
@@ -42,8 +44,7 @@ client = NeetoNotifications::Client.new(api_key: ENV["NN_API_KEY"])
 response = client.notification.send(
     device_token: "piowj03vnoi71p4mo20z0x",
     platform: "ios",
-    alert: "Hi User, Welcome to neetoChat",
-    body: { title: "Don't miss out on this!" }
+    body: { title: "Hello, Welcome to neetoChat", body: "Don't miss out on this!" }
```
Comment (chiraggshah) [inline README.md:47]: Can the parent key be `payload`? Because `body` has a `body` inside it which looks confusing.

**Review by chiraggshah — COMMENTED** [review]
@jaybhoyar _a : Please check the comment.

**Comment by jaybhoyar** [comment]
@chiraggshah _a Please review.

**Review by chiraggshah — APPROVED** [review]
Looks good.

**Pull request merged**